tests/test-cases/Group7-Regression/7-01-Regression.robot

```diff
@@ -29,3 +29,4 @@ Regression test
     ${ret}=  Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Run  govc vm.info %{VCH-NAME}
     Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Set Test Variable  ${out}  ${ret}
     Should Contain  ${out}  Photon - VCH
+    No Leaked Sessions  vic-machine
```

Comment:
Sorry, I meant - Docker-Util.robot/Run Regression Tests, which will ensure that it gets run in all of our tests that call that keyword, not just this test which only gets run on PRs.